images/linux/toolsets/toolset-1804.json

```diff
@@ -67,6 +67,7 @@
             "platform" : "linux",
             "arch": "x64",
             "versions": [
+                "*",
```

Comment:
you do not need a second asterisk here, it literally means "any version it finds" and it is used for the toolcache pester test mostly

Reply:
Ah, ok, thank you!

Reply:
Removed 👍

Reply:
Do you have any suggestions for how we could test that the Linux toolsets contain exactly 2 versions of CodeQL? We do it in a separate `Common.Tests` file for macOS — do you recommend reusing the same logic in the equivalent file for Linux?